tests/constants/urls.js

```diff
@@ -7,6 +7,7 @@ export const APPS = {
   PROFILE: 'https://my.realdevsquad.com',
   TASKS: 'https://my.realdevsquad.com/tasks',
   IDENTITY: 'https://my.realdevsquad.com/identity',
+  MY_STATUS: 'https://my.realdevsquad.com/',
```

Comment:
constant created already, please use that

Reply:
Done